### DISCUSSION TITLE
Error while installing mod_auth_openidc-2.4.11.2-1.el7.x86_64.rpm in Apache server

### DISCUSSION THREAD

**Discussion by Supraja1989** [OpenIDC/mod_auth_openidc#886] [Q&A] [answered]
Hi,

   Getting below error while installing the package. I have placed the dependent cjose & hiredis lib files in /usr/lib64 directoty.
yum install mod_auth_openidc-2.4.11.2-1.el7.x86_64.rpm
Error: Package: mod_auth_openidc-2.4.11.2-1.el7.x86_64 (/mod_auth_openidc-2.4.11.2-1.el7.x86_64)
           Requires: hiredis
Error: Package: mod_auth_openidc-2.4.11.2-1.el7.x86_64 (/mod_auth_openidc-2.4.11.2-1.el7.x86_64)
           Requires: libcjose.so.0()(64bit)
Error: Package: mod_auth_openidc-2.4.11.2-1.el7.x86_64 (/mod_auth_openidc-2.4.11.2-1.el7.x86_64)
           Requires: cjose
Error: Package: mod_auth_openidc-2.4.11.2-1.el7.x86_64 (/mod_auth_openidc-2.4.11.2-1.el7.x86_64)
           Requires: libhiredis.so.0.12()(64bit)
cd /usr/lib64
 ls -ltr | grep cjose
-rwxr-xr-x   1 root root      299072 Apr 27  2018 libcjose.so.0.0.0
lrwxrwxrwx   1 root root          17 Jul 21 15:38 libcjose.so.0 -> libcjose.so.0.0.0

ls -ltr | grep hire
-rwxr-xr-x   1 root root       54080 Feb 15  2020 libhiredis.so.0.12
lrwxrwxrwx   1 root root          18 Jul 21 15:36 libhiredis.so.0 -> libhiredis.so.0.12

Can somebody help in installing mod_auth_openidc.so in Linux servers

Regards
Supraja G

**Comment by zandbelt** [comment, marked as answer]
perhaps you installed hiredis /cjose from source? you'll need to install the dependencies as RPMs

**Reply by Supraja1989** [reply]
Hi,

    Thanks for the information. Now I have installed cjose from RPM which is available in the repository. I can see only below error now.

Error: Package: mod_auth_openidc-2.4.11.2-1.el7.x86_64 (/mod_auth_openidc-2.4.11.2-1.el7.x86_64)
           Requires: hiredis
Error: Package: mod_auth_openidc-2.4.11.2-1.el7.x86_64 (/mod_auth_openidc-2.4.11.2-1.el7.x86_64)
           Requires: libhiredis.so.0.12()(64bit)

Do we have hiredis rpm too available ??

Regards
Supraja G

**Reply by zandbelt** [reply]
you can get it here: https://download-ib01.fedoraproject.org/pub/epel/7/x86_64/Packages/h/hiredis-0.12.1-2.el7.x86_64.rpm

**Reply by Supraja1989** [reply]
Thanks much. Issue has been resolved now. Module has been installed successfully.
